react/features/base/dialog/components/native/PageReloadDialog.tsx

```diff
@@ -61,25 +59,14 @@ class PageReloadDialog extends Component<IPageReloadDialogProps, IPageReloadDial
 
         const timeoutSeconds = 10 + randomInt(0, 20);
 
-        let message, title;
-
-        if (this.props.isNetworkFailure) {
-            title = 'dialog.conferenceDisconnectTitle';
-            message = 'dialog.conferenceDisconnectMsg';
-        } else {
-            title = 'dialog.conferenceReloadTitle';
-            message = 'dialog.conferenceReloadMsg';
-        }
-
         this.state = {
-            message,
             timeLeft: timeoutSeconds,
-            timeoutSeconds,
-            title
+            timeoutSeconds
```

Comment:
Why store this in state, when it's never set? You can store it as an instance variable: this.timeoutSeconds.